docs/advanced-features/using-mdx.md

```diff
@@ -64,6 +64,8 @@ The following steps outline how to setup `@next/mdx` in your Next.js project:
      - package.json
    ```
 
+> **Note**: When installing `MDX` packages, you might get a warning message that requires webpack to be installed as a peer dependency. This can be ignored since Next.js already comes with webpack bundled in.
```

Comment:
Seems like we should fix this upstream if we can, potentially we can send a PR to `@mdx-js/loader` to make webpack an optional peer dependency or we could `ncc` it and provide our own for `@next/mdx` 🤔